Bottom: 21a48f88beb74a48228cd4e9c2a3118aad3d5383
Top:    2c5498f238584b081834b15773845a12f7240582
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 15:31:59 -0500

Refresh of buildbot-parameterize-env

---
diff --git a/docker/buildbot-master/buildbot/buildbot.tac b/docker/buildbot-master/buildbot/buildbot.tac
index c3b2e87..b87ce04 100644
--- a/docker/buildbot-master/buildbot/buildbot.tac
+++ b/docker/buildbot-master/buildbot/buildbot.tac
@@ -1,4 +1,5 @@
 import sys
+import os
 
 from twisted.application import service
 from twisted.python.log import FileLogObserver
@@ -6,7 +7,7 @@ from twisted.python.log import ILogObserver
 
 from buildbot.master import BuildMaster
 
-basedir = '/buildbot'
+basedir = os.env('BUILDBOT_DATA', '/buildbot')
 configfile = 'master.cfg'
 
 # note: this line is matched against to check that this is a buildmaster
